packages/react-theme/etc/react-theme.api.md

```diff
@@ -352,6 +376,21 @@ export type FontWeightTokens = {
     fontWeightSemibold: number;
 };
 
+// @public (undocumented)
+export type HorizontalSpacingTokens = {
+    spacingHorizontalNone: string;
+    spacingHorizontalXXS: string;
```

Comment:
@miroslavstastny I followed the figma naming for the spacing from the figma.  When I needed to add the spacingHorizontal and spacingVertical prefixes, they are a little hard to read.  Do you think we should have an underscore between the prefix and the size (e.g. spacingHorizontal_XXS) ?  Or maybe you have a better way to name them?